clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -2411,6 +2411,50 @@ public void testRemoveMembersFromGroup() throws Exception {
             assertNull(noErrorResult.all().get());
             assertNull(noErrorResult.memberResult(memberOne).get());
             assertNull(noErrorResult.memberResult(memberTwo).get());
+
+            // Return with success for "removeAll" scenario
```

Comment:
This test looks good, but it seems that we didn't test the case where some members get deleted successfully while some are not?

Reply:
Good catch! Added the test for partial failure